### PR TITLE
goof zombie chem recipe making sense

### PR DESCRIPTION
**Pull request by Agateophobia**
## About The Pull Request

sense me like and so liquid gibs make more sense than food poisoning for zombies

## Why It's Good For The Game

sense good 

## Changelog
:cl:
code: change one line so goof zombies have liquid gibs instead of food poisoning in recipe
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -736,13 +736,13 @@
 //	name = /datum/reagent/mutationtoxin/zombie
 //	id = /datum/reagent/mutationtoxin/zombie
 //	results = list(/datum/reagent/mutationtoxin/zombie = 1)
-//	required_reagents  = list(/datum/reagent/aslimetoxin = 1, /datum/reagent/toxin = 1, /datum/reagent/toxin/bad_food = 1) //Because rotting
+//	required_reagents  = list(/datum/reagent/aslimetoxin = 1, /datum/reagent/drug/krokodil = 10, /datum/reagent/liquidgibs = 10) //Because rotting and liquid gibs are more rotting than bad food
```
Comment (qwertyquerty) [inline code/modules/reagents/chemistry/recipes/others.dm]: Why did you change these commented out lines

Comment (Agateophobia) [reply code/modules/reagents/chemistry/recipes/others.dm]: I think I changed those by accident, gonna change it to original state

**Comment by qwertyquerty** [comment]
@agateophobia got the whole squad waiting

**Comment by Agateophobia** [comment]
I was actually havin little renovation in my room and I didn't want to get pneumoconiosis
https://gyazo.com/e8071097ebe8a8658e0ab5601e6e4712

**Comment by Agateophobia** [comment]
oh god oh no i'm a smoothbrain

**Comment by Agateophobia** [comment]
i'll just make a new one.
